### PR TITLE
Feat/15 create main page ee

### PR DESCRIPTION
**Pull request by Emimint**
Here is the front page.

I am not sure about the font unit size or overall disposition of the elements, so please let me know if you think I should fix anything!

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+  background-color: white;
+}
+
+#enter {
+  border: 2px black solid;
+  margin: 1rem;
+  font-size: larger;
+  padding: 1rem;
+  width: 40vw;
+}
```
Comment (terrifricker) [inline style.css:64]:
```suggestion
  background-color: white;
}
```

File: style.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+html {
+  color: #bb4e1e;
+  display: table;
+  margin: auto;
+}
+
+h1 {
+font-family: 'Bantangas', "DM Sans-Regular", sans-serif;
+font-size: 5rem;
```
Comment (terrifricker) [inline style.css:24]:
```suggestion
font-size: 3rem;
```

File: style.css [hunk 3]
```diff
@@ -0,0 +1,64 @@
...
+font-weight:bolder;
+font-style:normal;
+}
+
+h2 {
+font-family: "DM Sans-Regular", sans-serif;
+font-size: 4rem;
```
Comment (terrifricker) [inline style.css:31]:
```suggestion
font-size: 2.25rem;
```

File: style.css [hunk 4]
```diff
@@ -0,0 +1,64 @@
+@font-face {
+  font-family: "Bantangas";
+  src: url("Fonts/Batangas/Web Fonts/Batangas-Bold.woff");
+  src: url("Fonts/Batangas/Web Fonts/Batangas-Bold.woff2");
```
Comment (terrifricker) [inline style.css:4]: Switch these two, so woff2 is used by browser if possible (newer).
```suggestion
  src: url("Fonts/Batangas/Web Fonts/Batangas-Bold.woff2");
  src: url("Fonts/Batangas/Web Fonts/Batangas-Bold.woff");
  ;
```

File: style.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+}
+@font-face {
+  font-family: "DM Sans-Regular";
+  src: url("Fonts/DM_Sans/DMSans-Regular.ttf");
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
```
Comment (terrifricker) [inline style.css:13]: I usually put this on body instead of the global object.
```suggestion

```

File: style.css [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+}
+
+html {
+  color: #bb4e1e;
+  display: table;
```
Comment (terrifricker) [inline style.css:18]: If you are using flex, this isn't needed.
```suggestion

```

File: style.css [hunk 7]
```diff
@@ -0,0 +1,64 @@
...
+font-style:normal;
+color: black;
+}
+
+body {
+  background-image: url("/Images/norris-background.jpg");
+  background-attachment: fixed;
+  background-position: center;
+  background-repeat: no-repeat;
+  -webkit-background-size: cover;
```
Comment (terrifricker) [inline style.css:42]: I don't think -webkit is needed anymore for background size.
```suggestion

```

**Review by terrifricker — COMMENTED** [review]
It's looking great Emilie!  I do think your fonts are a little big.  I've suggested different sizes as well as a few other items.

File: style.css [hunk 8]
```diff
@@ -0,0 +1,64 @@
...
+
+h2 {
+font-family: "DM Sans-Regular", sans-serif;
+font-size: 4rem;
+font-weight:bolder;
+font-style:normal;
+color: black;
+}
+
+body {
```
Comment (terrifricker) [inline style.css:37]:
```suggestion
body {
  margin: 0;
```

**Comment by terrifricker** [comment]
I also think we can delete some of the font files that aren't used.

**Review by QuangLyHo — APPROVED** [review]

**Review by alekzandriia — APPROVED** [review]

**Pull request merged**